src/lib/ldap/base.h

```diff
@@ -319,7 +319,7 @@ typedef struct {
  * Used to store the array of attributes we'll be querying for.
  */
 typedef struct {
-	map_t const *maps;				//!< Head of list of maps we expanded the RHS of.
+	fr_map_list_t	const *maps;			//!< Head of list of maps we expanded the RHS of.
```

Comment:
Not sure this should be a pointer?

Reply:
It's assigned in fr_ldap_map_expand() - which is passed a pointer - so yes.

Reply:
Ah you're right.  I'll double check this isn't the case with the rest.